libs/component-library/src/lib/progress-bar/progressBar.tsx

```diff
@@ -0,0 +1,55 @@
+import { useTheme } from '@shopify/restyle';
+import React from 'react';
+import { LayoutAnimation } from 'react-native';
+
+import { FxBox, FxBoxProps } from '../box/box';
+import { FxTheme } from '../theme/theme';
+import { WINDOW_WIDTH } from '../utils/constants';
+
+export type FxProgressBarProps = {
+  progress: number;
+  width?: number;
+  total?: number;
+  disabled?: boolean;
+} & FxBoxProps;
+
+const FxProgressBar = ({
+  width,
+  progress,
+  height = 4,
+  total = 100,
+  ...rest
+}: FxProgressBarProps) => {
+  const { spacing } = useTheme<FxTheme>();
+  const _width = width || WINDOW_WIDTH - 2 * spacing[20];
+  const _progress = Math.min(Math.max(0, progress), total);
+  const progressWidth = Math.round(_width * (_progress / total));
+
+  React.useEffect(() => {
+    LayoutAnimation.configureNext({
```

Comment:
There's a utility function in `utils/animations.ts` we can use here

Reply:
Have updated